### PR TITLE
Us05 fil actu

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

File: frontend/src/components/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import "./style/NavBar.scss";
+
+class NavBar extends Component {
```
Comment (LoicBrassart) [inline frontend/src/components/NavBar.jsx:4]: Seems to me this one could have been a function, don't you think ? 
(That's not a blocker, just a question)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- Font files have nothing to do in `src`, please move them to `public`
- `reset.css` has been moved and renamed (with a capital letter !), why is this ?

**Review by Gumatan — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Gumatan — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**